ssha/ssm.py

```diff
@@ -81,6 +87,41 @@ def find_instances():
     return instance_info_list
 
 
+def generate_key():
+    path = os.environ['HOME'] + "/.ssh/ssha/"
+    key_path = path + "id_rsa_{}".format(time.time())
+
+    if not os.path.exists(path):
+        os.mkdir(path)
+
+    os.system('ssh-add -d {}*'.format(path))
```

Comment:
Do we definitely want to remove old keys from the agent? As long as the files get deleted that might be enough. There is also a `ssh-add -t life` option which might be of interest if we do want to remove old keys.